module/users/helper.js

```diff
@@ -467,6 +467,65 @@ module.exports = class UsersHelper {
           });
         }
 
+        let totalCount = 0;
+        let mergedData = [];
+        //fetch all targeted programs to users
+        let targetedPrograms = await programsHelper.forUserRoleAndLocation(
```

Comment:
@ankitpws Please ensure we follow the checklist, this and the survey service PR are missing a checklist item - "When calling another function inside a given function, add comments explaining the purpose and meaning of the passed arguments and expected return values."

Reply:
Added request and response as well